pelicanconf.py

```diff
@@ -17,7 +17,8 @@
 TIMEZONE = 'Europe/Prague'
 
 DEFAULT_LANG = 'en'
-DATE_FORMATS = {'en': ('en_US', '%b %d %Y')}
+LOCALE = ('en_US', 'usa')
+DATE_FORMATS = {'en': '%b %d %Y'}
```

Comment:
I'm not sure about the implications here. Would this work as well for you (with `import sys` on the top)?

```python
if sys.platform == 'win32':
    DATE_FORMATS = {'en': ('usa', '%b %d %Y')}
else:
    DATE_FORMATS = {'en': ('en_US', '%b %d %Y')}
```

Reply:
Integrated in 1d8e425400aca0ea1834c2d980cc49a27e81fc0a.